src/screens/SendSheet.js

```diff
@@ -983,19 +987,34 @@ export default function SendSheet(props) {
             watchedAccounts={watchedAccounts}
           />
         )}
-        {showAssetList && (
-          <SendAssetList
-            hiddenCoins={hiddenCoinsObj}
-            nativeCurrency={nativeCurrency}
-            network={network}
-            onSelectAsset={sendUpdateSelected}
-            pinnedCoins={pinnedCoinsObj}
-            savings={savings}
-            sortedAssets={sortedAssets}
-            theme={theme}
-            uniqueTokens={sendableUniqueTokens}
-          />
-        )}
+        {showAssetList &&
+          (!isEmptyWallet ? (
+            <SendAssetList
+              hiddenCoins={hiddenCoinsObj}
+              nativeCurrency={nativeCurrency}
+              network={network}
+              onSelectAsset={sendUpdateSelected}
+              pinnedCoins={pinnedCoinsObj}
+              savings={savings}
+              sortedAssets={sortedAssets}
+              theme={theme}
+              uniqueTokens={sendableUniqueTokens}
+            />
+          ) : (
+            <View
```

Comment:
doing this to center `NoResults` on the screen bc the bottom of the sheet actually extends far below the bottom of screen, even though it's not scrollable 🤔 .